<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesGeoreference</class>
 <widget class="QWidget" name="PreferencesGeoreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Geocoding">
     <property name="title">
      <string>Dive site geo lookup</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QCheckBox" name="enable_geocoding">
        <property name="text">
         <string>Enable geocoding for dive site management</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="parse_without_gps">
        <property name="text">
         <string>Parse site without GPS data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tag_existing_dives">
        <property name="text">
         <string>Same format for existing dives</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="title">
      <string>Dive Site Layout</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QComboBox" name="first_item">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_29">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="second_item">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="third_item">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>109</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
